index.js

```diff
@@ -165,7 +166,9 @@ class WritableState {
     while ((stream._duplexState & WRITE_STATUS) === WRITE_QUEUED) {
       const data = this.shift()
       stream._duplexState |= WRITE_ACTIVE_AND_SYNC
-      stream._write(data, this.afterWrite)
```

Comment:
`callPromise(this, stream._write(data), this.afterWrite, this.afterWriteError)` ?